core/store/src/columns.rs

```diff
@@ -300,6 +300,12 @@ pub enum DBCol {
     /// - *Rows*: only one key with 0 bytes.
     /// - *Column type*: `EpochSyncProof`
     EpochSyncProof,
+    /// Mapping of ShardUId to the underlying ShardUId database key prefix for the State column.
+    /// It could be parent shard after resharding, ancestor shard in case of many resharding,
+    /// or just map shard to itself if there was no resharding or we updated the mapping during state sync.
```

Comment:
wording nit (unformatted):
```suggestion
    /// Mapping of ShardUId to the ShardUId that should be used as the database key prefix for the State column.
    /// The mapped ShardUId value can be the parent shard after resharding, an ancestor shard after many resharding
    /// or just map shard to itself if there was no resharding or the mapping was removed after node stopped tracking the shard. 
```

Reply:
fixed in https://github.com/near/nearcore/pull/12084/commits/fee1c6ae69a756976890a48d2ceaefa721943204